lib/cocina/models/dro_access.rb

```diff
@@ -4,6 +4,8 @@ module Cocina
   module Models
     class DROAccess < Struct
       attribute :access, Types::Strict::String.default('dark').enum('world', 'stanford', 'location-based', 'citation-only', 'dark').meta(omittable: true)
```

Comment:
do we need a new value here for cdl?

Reply:
No, because CDL is separate from the access value.  Which would continue to be `stanford`